doc/source/testing.rst

```diff
@@ -106,8 +136,42 @@ Linting is performed by `Flake8`_.
 
 .. code-block:: bash
 
-    $ tox -e $(tox -l | grep lint | paste -d, -s -)
+    $ tox -e lint
 
 .. _`Flake8`: http://flake8.pycqa.org/en/latest/
 
+
+Documentation
+-------------
+
+Create the documentation, using `sphinx`_.
+
+.. code-block:: bash
+
+    $ tox -e doc
+
+.. _`sphinx`: http://www.sphinx-doc.org
+
+Metadata validation
+-------------------
+
+Check packaging informations, using `checkdocs`_ and `twine`_.
```

Comment:
```suggestion
Check packaging metadata information, using `checkdocs`_ and `twine`_.
```

Reply:
I guess DCO will be missing if I commit from github GUI?

Reply:
Worst-case, I think you can add it manually when the commit is made. As far as I know there's no special metadata associated with the signoff, it's just a line appended to the commit message.

Reply:
rephrased.